mixer/pkg/attribute/list.gen.go

```diff
@@ -3,14 +3,14 @@
 
 package attribute
 
-func GlobalList() ([]string) { 
-    tmp := make([]string, len(globalList))
-    copy(tmp, globalList)
-    return tmp
+func GlobalList() []string {
```

Comment:
not sure why this file is changing here?

Reply:
This file keeps sneaking in everytime I do a source format. Removed.